### PR TITLE
Update to v1.6

### PR DESCRIPTION
**Pull request by daisieh**
This just updates the version numbers for Maven, including the README. If merged, can you update the version stored at the maven repository?

### CODE REVIEW

**Comment by rvosa** [comment]
Hi Daisie,

just to verify, are you sure that the new maven naming scheme is correct?
Specifically, does the README.md in the root of the repository indeed
reflect our intentions? The reason I'm asking is that in the previous
version, the naming was like this:

   <groupId>org.nexml.model</groupId>
  <artifactId>nexml</artifactId>
  <version>1.5-SNAPSHOT</version>

And this has changed to:

      <groupId>org.nexml</groupId>
      <artifactId>model</artifactId>
      <version>1.6</version>

In other words, both the groupId namespace have changed
(s/org.nexml.model/org.nexml/), as well as the artifactId (s/nexml/model/),
which results in a different folder structure inside the maven repository
and a different, less descriptive, name for the nexml JAR. Why would we
want it this way? There could very well be reasons I'm not aware of, just
checking to make sure.

Best wishes,

Rutger

**Comment by daisieh** [comment]
I refactored the targets in the last PR #19 so that there are now several separate target products instead of one single uber-NeXML jar. The rearranged folder structure reflects that, and so do the artifactIds. It seems to me to be good practice to allow people to be able to install just one target, like the model, without having to build dependencies on all of them.

So in the new arrangement, there are four separate sub-targets that are built by the root POM, and any one of those should be able to be used as its own dependency. These are model, mesquite-module, transformer, and validator. I assumed that the "model" part of the groupId was really part of the model target, but perhaps I'm misunderstanding your intent?

**Comment by rvosa** [comment]
Ah, great, that makes sense.

So, I should actually make all these artifacts separately and push them all
into the maven repo, yes? At this point, all maven knowledge has leaked out
of my brain so I am literally back to 'hello world' as far as that is
concerned (so, any hints?).

Thanks! :)

**Comment by daisieh** [comment]
I think separate artifacts, yes. I have never made a maven repo either; I was hoping you knew how! Time for some googling, I guess :)

**Comment by daisieh** [comment]
Hi Rutger, sorry to bug you again after several months...did you figure out how to update the maven repo, or should I look into it myself?

**Comment by rvosa** [comment]
Hi Daisie,

recently I have been working on resurrecting the TreeBASE build, which uses
the 1.5-SNAPSHOT version of the NeXML libraries. To this end I revisited
the maven repo, and I now know that this is hosted here:
https://github.com/nexml/nexml.github.io/tree/master/maven/repository/org/nexml/model/nexml/1.5-SNAPSHOT

Which means it becomes available thusly in the pom.xml of TreeBASE:

<repository>
<id>m2.nexml.repos</id>
<name>NeXML Remote Repository</name>
<url>http://nexml.github.io/maven/repository</url>
</repository>



<dependency>
<groupId>org.nexml.model</groupId>
<artifactId>nexml</artifactId>
<version>1.5-SNAPSHOT</version>
</dependency>

Now, it appears to me that your refactoring should be able to live
peacefully next to that. The way to do that would consist of something like:

- compile the nexml artifacts
- make maven create a local repo and install your nexml artifacts in it
- clone the nexml.github.io repo website repo
- copy the folder structure directly around your nexml artifacts into the
right location in the website repo
- git add / commit / push

The exact steps were a bit of trial and error for me to get the metadata
right but eventually the build picked up the 1.5-SNAPSHOT artifacts. Maybe
it's best if you do the same trial and error for the mesquite build? I've
given you admin rights to the nexml.github.io repo so you can poke around
till it works.

Best wishes,

Rutger

**Comment by daisieh** [comment]
Thanks for the access! I can figure out how to set up the version I need for Mesquite.

I think that generally speaking, you are probably supposed to put a non-snapshotted version into the repo itself? Like, snapshots are meant to be temporary versions that you are building in your local repo.

**Comment by rvosa** [comment]
> Thanks for the access! I can figure out how to set up the version I need
> for Mesquite.
>
> I think that generally speaking, you are probably supposed to put a
> non-snapshotted version into the repo itself? Like, snapshots are meant to
> be temporary versions that you are building in your local repo.
>

I know, but I am an idiot and now this artifact version name has taken on a
life of its own. I should fix that. :)
